MathPortal/MathPortal/Source/Custom/Custom Views/ArrowView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  ArrowView.swift
+//  MathPortal
+//
+//  Created by Petra Čačkov on 26/12/2019.
+//  Copyright © 2019 Petra Čačkov. All rights reserved.
+//
+
+import UIKit
+
+class ArrowView: UIView {
+
+    private var fromView: EquationView = .Nil
+    private var toView: EquationView = .Nil
+    
+    private var arrowPath: UIBezierPath = UIBezierPath(ovalIn: .zero)
+
+    private lazy var arrowLength: CGFloat = { return  20 * scale }()
+    private lazy var arrowTipFrame: CGRect = { return CGRect(x: 0, y: 0, width: 5*scale, height: 5*scale) }()
+    
+    private(set) var offset: CGFloat = 0
+    private let spacing: CGFloat = 2
+    
+    var strokeColor: UIColor = .black
+    var scale: CGFloat = 1
+    
+    init(from fromView: EquationView, to toView: EquationView, scale: CGFloat) {
+        super.init(frame: .zero)
+        self.fromView = fromView
+        self.toView = toView
+        self.offset = max(fromView.verticalOffset, toView.verticalOffset)
+        self.scale = scale
+        refresh()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func refresh() {
+        addViews()
+        drawArrow()
+    }
+    
+    private func drawArrow() {
+        guard let fromViewBounds = fromView.view?.bounds else { return }
+    
+        arrowPath.move(to: CGPoint(x: fromViewBounds.width + spacing , y: offset))
```

Comment:
Missing a cleanup. Every time this method is called the `arrowPath` becomes bigger.